### PR TITLE
feat: add jsonpath and isolate tests

### PR DESCRIPTION
**Pull request by thomas9911**

### CODE REVIEW

File: functions/jsonpath-value/1.0/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import jsonpath from './jsonpath.min';
+
+const jsonpathImplementation = async ({ data, path, outputType }) => {
```
Comment (emiel-betty) [inline functions/jsonpath-value/1.0/index.js:3]: `jsonpathImplementation` is a kind of weird naming imo

File: functions/jsonpath-value/1.0/function.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        },
+        "allowedKinds": ["STRING"]
+      }
+    },
+    {
+      "info": "the jsonpath",
+      "name": "path",
+      "label": "Jsonpath",
+      "meta": {
+        "type": "Value",
```
Comment (emiel-betty) [inline functions/jsonpath-value/1.0/function.json:27]: Can this not be of type `Text`?

File: functions/jsonpath-value/1.0/function.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    "name": "ActionsIcon",
+    "color": "Orange"
+  },
+  "options": [
+    {
+      "info": "The actual data; can be text",
+      "name": "data",
+      "label": "Data",
+      "meta": {
+        "type": "Value",
```
Comment (emiel-betty) [inline functions/jsonpath-value/1.0/function.json:15]: Can this not be of type `Text`?

File: blocks/jsonpath.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": ["jsonpath"],
```
Comment (emiel-betty) [inline blocks/jsonpath.json:2]: You don't use a npm package, you should include the utils folder in the `includes` instead.

File: functions/jsonpath-value/1.0/index.js (hunk 1 above)
Comment (thomas9911) [reply functions/jsonpath-value/1.0/index.js:3]: how should i call it? jsonpathFunction ? or jsonpathActionStep ?

Comment (thomas9911) [reply functions/jsonpath-value/1.0/index.js]: Now directly set it as the default export (so i dont need to set a name :D )

File: functions/jsonpath-value/1.0/function.json [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    "name": "ActionsIcon",
+    "color": "Orange"
+  },
+  "options": [
+    {
+      "info": "The data or object",
+      "name": "data",
+      "label": "Data",
+      "meta": {
+        "type": "Text",
```
Comment (emiel-betty) [inline functions/jsonpath-value/1.0/function.json:15]: I think we should change this to type `Value` with the following kinds: JSON, OBJECT STRING?

**Review by emiel-betty — APPROVED** [review]

**Pull request merged**